protocols/medco/deterministic_switching_test.go

```diff
@@ -13,7 +13,6 @@ import (
 )
 
 func TestDeterministicSwitching(t *testing.T) {
-	t.Skip("Skipping due to error in medco: https://github.com/dedis/cothority/issues/479")
```

Comment:
Nope as for me it's deterministically failing I don't know why, I told @froelich to look into it so I keep it skipped